contracts/pcv/morpho/MorphoCompoundPCVDeposit.sol

```diff
@@ -151,6 +167,14 @@ contract MorphoCompoundPCVDeposit is PCVDeposit, ReentrancyGuard {
             amount
         );
 
+        int256 endingRecordedBalance = balance().toInt256();
```

Comment:
why not read the state in `lastRecordedBalance` ? it's freshly updated with _recordPNL() etc

Reply:
lastRecordedBalance is off by a hair because on deposit, you take a loss when compound rounds down in favor of their protocol, so you have to read balance instead to get the correct amount of balance in the system.